kevm-pyk/src/kevm_pyk/__main__.py

```diff
@@ -299,6 +299,50 @@ def exec_prove(
     sys.exit(failed)
 
 
+def exec_show_kcfg(
+    definition_dir: Path,
+    save_directory: Path,
+    spec_file: Path,
+    includes: List[str],
+    claim_label: Optional[str] = None,
+    spec_module: Optional[str] = None,
+    md_selector: Optional[str] = None,
+    nodes: Iterable[str] = (),
+    node_deltas: Iterable[Tuple[str, str]] = (),
+    to_module: bool = False,
+    minimize: bool = True,
+    **kwargs: Any,
+) -> None:
+    kevm = KEVM(definition_dir)
+
+    _LOGGER.info(f'Extracting claims from file: {spec_file}')
+    claim = single(
+        kevm.get_claims(
+            spec_file,
+            spec_module_name=spec_module,
+            include_dirs=[Path(i) for i in includes],
+            md_selector=md_selector,
+            claim_labels=([claim_label] if claim_label is not None else None),
+        )
+    )
+
+    kcfg = KCFGExplore.read_cfg(claim.label, save_directory)
+    if kcfg is None:
+        raise ValueError(f'Could not load CFG {claim} from {save_directory}')
```

Comment:
These lines are a copy/paste from exec_view_kcfg. Is it worth pulling some of it out into a helper?

Reply:
Done, Made a helper called `get_cfg_for_spec` in the `utils.py` module.